src/main/java/lrgs/gui/LrgsBuild.java

```diff
@@ -19,8 +19,8 @@
 */
 public class LrgsBuild
 {
-	public static final String buildDate = "@DATE@";
-	public static final String rcNum = "@RCNUM@";
-	public static final String version = "@VERSION@";
+	public static final String buildDate = "7.0.13";
```

Comment:
Is this, setting specific values, intentional?

Reply:
Yeah. Though certainly not ideal.